docs/core/compatibility/3.0.9-3.0.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Breaking changes, version 3.0 Preview 9 to 3.0 - .NET Core
+description: Lists the breaking changes from version 3.0 Preview 9 to version 3.0 Preview 8 of .NET Core, ASP.NET Core, and EF Core.
+ms.date: "09/10/2019"
+author: "rpetrusha"
+ms.author: "ronpet"
+---
+# Breaking changes for migration from Version 3.0 Preview 9 to 3.0
+
+> [!IMPORTANT]
+> This article is under construction. This is not a complete list of .NET Core breaking changes. For more information on .NET Core breaking changes, you can examine individual [breaking changes issues](https://github.com/dotnet/docs/issues?q=is%3Aissue+is%3Aopen+label%3Abreaking-change) in the dotnet/docs repo on GitHub. 
+
+If you are migrating from 3.0 Preview 9 to 3.0 of .NET Core, review the following topics for breaking changes that may affect your app:
+
+## Windows Forms
+
+[!INCLUDE[Switch.System.Windows.Forms.AllowUpdateChildControlIndexForTabControls compatibility switch not supported](~/includes/core-changes/windowsforms/deprecate-allowupdatechildcontrolindexfortabcontrols.md)]
+
+[!INCLUDE[Switch.System.Windows.Forms.EnableVisualStyleValidation compatibility switch not supported](~/includes/core-changes/windowsforms/deprecate-enablevisualstylevalidation.md)]
```

Comment:
wondering if it would look nice (or not) to have an horizontal rule in between changes. Right now it's hard to see where they begin and end.

```suggestion
***

[!INCLUDE[Switch.System.Windows.Forms.EnableVisualStyleValidation compatibility switch not supported](~/includes/core-changes/windowsforms/deprecate-enablevisualstylevalidation.md)]
```